### PR TITLE
Wulumunusha Adjustments

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Max druggy is raised, it no longer silences you for a legitimately crazy amount of time.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!bugfix

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drugs.dm [hunk 1]
```diff
@@ -327,5 +327,5 @@
 	fallback_specific_heat = 1
 
 /datum/reagent/wulumunusha/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
-	M.druggy = max(M.druggy, 50)
-	M.silent += 15
\ No newline at end of file
+	M.druggy = max(M.druggy, 100)
+	M.silent += 5
```
Comment (Alberyk) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drugs.dm]: Could also add a max silence, I guess.

Comment (ParadoxSpace) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drugs.dm]: thanks for the suggestion

**Pull request merged**